### PR TITLE
 assert roots flag

### PR DESCRIPTION
**Pull request by fcollman**
This is a mirror to #255 to add the same flag to the api, and normalize the behavior between the two versions.

### CODE REVIEW

**Comment by fcollman** [comment]
I changed the flag to default to False, as this more closely mirrors the present behavior of master, but now one can set the flag to true if you want to be strict about it, and now the backend api also has the option in master via #257 though it presently is not being used.

**Comment by fcollman** [comment]
default False on the rest-api also means that no changes are needed in cloud-volume to fix the 503 errors you get from some cutout calls with remapping turned on.

File: pychunkedgraph/backend/chunkedgraph.py [hunk 1]
```diff
@@ -2432,6 +2433,11 @@ def get_roots(self, node_ids: Sequence[np.uint64],
                 return parent_ids
             else:
                 time.sleep(0.5)
+        # if we have failed to return by now, then we we have some roots
+        # that have failed to get to the stop layer as if we want to assert_roots
+        # then we should fail that assertion
+        if assert_roots:
+            assert False
```
Comment (sdorkenw) [inline pychunkedgraph/backend/chunkedgraph.py:2440]: This could just be `assert not assert_roots` or even better an exception :)

Comment (akhileshh) [reply pychunkedgraph/backend/chunkedgraph.py:2440]: Exception with message is better, `assert not assert_roots` can be confusing

Comment (fcollman) [reply pychunkedgraph/backend/chunkedgraph.py]: Fixed

**Pull request merged**